src/Mvc/Mvc.Core/src/ModelBinding/DefaultModelBindingContext.cs

```diff
@@ -1,6 +1,8 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+#nullable enable
```

Comment:
When can these be replaced by enabling globally in the project?

Reply:
There's still a ton of files in MVC.Core that aren't nullable enabled. Maybe a couple more iterations before we can start enabling it in the project versus individual files.